header-bar/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "header-bar",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "bootstrap": "^3.3.7",
+    "react": "^15.6.1",
+    "react-bootstrap": "^0.31.1",
+    "react-dom": "^15.6.1",
+    "react-router": "^4.1.1",
+    "react-router-dom": "^4.1.1",
+    "url-loader": "^0.5.9"
+  },
+  "devDependencies": {
+    "react-scripts": "1.0.10"
+  },
```

Comment:
I used "create-react-app" to initialize this. CRA uses npm but i've been learning how react is better implemented into a full MERN app and will adjust accordingly to use nodemon.